packages/upload-client/src/blob.js

```diff
@@ -303,7 +304,46 @@ export async function add(
     })
   }
 
-  return multihash
+  // Ensure the blob has been accepted
+  const acceptReceipt = await new ReceiptPoller(options).poll(
+    nextTasks.accept.task.link()
+  )
+
+  // @ts-ignore Property
+  /* c8 ignore next 5 */
+  if (!acceptReceipt?.out.ok?.site) {
+    throw new Error(`failed ${BlobCapabilities.add.can} invocation`, {
+      cause: 'failed to get blob/accept receipt',
+    })
+  }
```

Comment:
nit: would be good to distinct this error, from Not Found, or Other errors

Reply:
addressed